### ISSUE TITLE
Example notebooks should import torchvision.models as a different name

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#94]
In the example notebooks, torchvision.models is getting imported as "models" but then that gets overwritten later on by the "models" dictionary, and that is confusing.

**Comment by KevinMusgrave** [comment]
Fixed
